.github/workflows/ci.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
       matrix:
         features:
           - "sync-secret-service"
+          - "sync-secret-service,linux-native"
```

Comment:
This additional test is a good one, and needs to be part of the CI once #215 (or #220) is integrated.  Thanks for pointing that out!